common/bin/oo-diagnostics

```diff
@@ -1226,6 +1228,28 @@ class OODiag
       do_warn "Using a self-signed certificate for the broker"
     end
 
+    # Retrieve the SSL cert from broker_host_env
+    env_response = `curl -k -s -v https://#{broker_host_env.chomp}/ -o /dev/null 2>&1`
+    if $? != 0
+      do_fail <<-CANNOTCONNECT
+        Attempt to connect to #{broker_host_env} but failed. curl returned with
+        #{env_response.strip.empty? ? "no output." : "the following output:\n" + env_response}
+
+        Please check whether the httpd service in #{broker_host_env} is running.
+      CANNOTCONNECT
+      return
+    end
+
+    # Get ssl cert's issuer, subject and commonname
+    env_issuer = env_response.slice(/^\*\s*issuer: .*$/).gsub(/^\*\s*issuer: /,"")
```

Comment:
Might be more elegant to use `env_response.scan(/^\*\s*issuer: (.*)$/).first` or `/^\*\s*issuer: (.*)$/.match(env_response).first`.